lib/std/array_list.zig

```diff
@@ -16,6 +16,10 @@ pub fn ArrayList(comptime T: type) type {
     return ArrayListAligned(T, null);
 }
 
+/// A contiguous, growable list of byte-aligned items in memory.
+/// This is a wrapper around an array of T values aligned to `alignment`-byte
+/// addresses. If the specified alignment is `null`, then `@alignOf(T)` is used.
+/// Initialize with `init`.
```

Comment:
Mention that it stores an allocator, and that the unmanaged version can be used for one that accepts an allocator parameter for relevant functions.

Reply:
done